exercises/concept/calculator-conundrum/src/test/CalculatorConundrumTest.java

```diff
@@ -0,0 +1,84 @@
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class CalculatorConundrumTest {
+
+    @Test
+    public void additionWithSmallOperands() {
+        assertEquals("22 + 25 = 47", new CalculatorConundrum(22, 25, "+").calculate());
```

Comment:
Throughout - we are generally prefering the AssertJ assertions:

```
assertThat(new CalculatorConundrum(22, 25, "+").calculate()).isEqualTo("22 + 25 = 47");
```

Ones that throw exceptions should still use assertThrows.